pkg/cmd/server/start/master_args.go

```diff
@@ -82,6 +86,7 @@ func BindMasterArgs(args *MasterArgs, flags *pflag.FlagSet, prefix string) {
 	flags.Var(&args.DNSBindAddr, prefix+"dns", "The address to listen for DNS requests on.")
 	flags.BoolVar(&args.PauseControllers, prefix+"pause", false, "If true, wait for a signal before starting the controllers.")
 
+	flags.StringVar(&args.ClusterDomain, prefix+"cluster-domain", "cluster.local", "The authoritative domain of the DNS server.")
```

Comment:
I still don't think this is commonly changed enough to warrant flag exposure. @smarterclayton, wdyt?